Bug! Module not found: Can't resolve './Dropdown'

Hello, thanks for making this fork of `react-dropdown`.

I was using version `1.0.3` and everything was working fine, now the latest version is causing this error:

```
~/myRepo/node_modules/react-dropdown-now/dist/index.js
Module not found: Can't resolve './Dropdown' in '~/myRepo/node_modules/react-dropdown-now/dist'
```

Please help.

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     }
   ],
   "files": [
-    "dist/index.js",
+    "dist/*",
     "index.d.ts",
     "style.css"
   ],

